addons/binding/org.openhab.binding.hdanywhere/ESH-INF/thing/matrix.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hdanywhere"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- HDanywhere Thing Type -->
+    <thing-type id="matrix">
```

Comment:
Valid point, but 8x8 is the maximum they have for now. It did it that way because part of their product line is sold as bare racks + modules you can insert, so one could end up with  any combination of ports on the ingress or egress of the device